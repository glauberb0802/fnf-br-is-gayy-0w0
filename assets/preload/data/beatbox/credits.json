{
	"music": [
		"kiwiquest"
	],
	"arter": [
		"Lex","Headdzo"
	],
	"coder": [
		"Data"
	],
	"charter": [
		"Brooklyn"
	]
}