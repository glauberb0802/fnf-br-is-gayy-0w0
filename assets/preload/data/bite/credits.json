{
	"music": [
		"kiwiquest"
	],
	"arter": [
		"Binejyeah","mr DJ", "headdzo"
	],
	"coder": [
		"fabr"
	],
	"charter": [
		"Rotty"
	]
}