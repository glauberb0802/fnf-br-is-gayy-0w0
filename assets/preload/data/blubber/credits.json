{
	"music": [
		"emihead"
	],
	"arter": [
		"Kaszper","libur","Headdzo"
	],
	"coder": [
		"cold_vee"
	],
	"charter": [
		"Rotty"
	]
}