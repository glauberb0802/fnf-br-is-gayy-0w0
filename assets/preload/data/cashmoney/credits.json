{
	"music": [
		"Smokey99k"
	],
	"arter": [
		"sNoob","Libur","Loss"
	],
	"coder": [
		"Data"
	],
	"charter": [
		"Brooklyn","gibz679"
	]
}