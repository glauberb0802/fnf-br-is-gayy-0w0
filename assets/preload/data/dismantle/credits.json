{
	"music": [
		"Marstarbro"
	],
	"arter": [
		"loss","cold_vee","libur"
	],
	"coder": [
		"cold_vee"
	],
	"charter": [
		"Rotty"
	]
}