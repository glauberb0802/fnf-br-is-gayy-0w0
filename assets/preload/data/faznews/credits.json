{
	"music": [
		"kiwiquest"
	],
	"arter": [
		"QuietTomato"
	],
	"coder": [
		"data"
	],
	"charter": [
		"Rotty"
	]
}