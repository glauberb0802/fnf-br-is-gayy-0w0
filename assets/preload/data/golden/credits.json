{
	"music": [
		"Zeroh"
	],
	"arter": [
		"Jonspeedarts","Loss"
	],
	"coder": [
		"cold_vee"
	],
	"charter": [
		"Rotty"
	]
}